README.md

```diff
@@ -63,52 +58,27 @@ on how you can run your own `go-quai` instance.
 By far the most common scenario is people wanting to simply interact with the Quai
 network: create accounts; transfer funds; deploy and interact with contracts. For this
 particular use-case the user doesn't care about years-old historical data, so we can
-fast-sync quickly to the current state of the network. To do so:
+fast-sync quickly to the current state of the network. 
```

Comment:
full-sync, there is no fast-sync at the moment

Reply:
This paragraph is removed because we removed the other functionalities as well